### ISSUE TITLE
PFS8  Payment couldn't be completed because: there is no route available

### ISSUE THREAD

**Issue by agatsoh** [raiden-network/raiden#5357]
[pfs8_mediator_goes_offline_2019_11_24.tar.gz](https://github.com/raiden-network/raiden/files/3886212/pfs8_mediator_goes_offline_2019_11_24.tar.gz)
## Problem Definition
Scenario in a nutshell
node0 : `0x43b6Bb8E213496e9D1a4E0A3753AfbF339953DFA`
node1 : `0x1B5ca29302e39d2B2794De3CCFE783FFeC56db45`
node2 : `0x57C27B4D402B80D019C447D82721Fa1C9EdcFB12`
node3 : `0x1c5E3b08b928A75585fe35286F8a3C89f417ba93`
node4 : `0x607aFA89CC0248Ac8e139eBde26F3e9247d4C7c6`

Scenario logs:

```
{
  "task": "<TransferTask: {'from': 0, 'to': 3, 'amount': 1000000000000000, 'expected_http_status': 200}>",
  "event": "Task errored",
  "logger": "scenario_player.tasks.base",
  "level": "error",
  "timestamp": "2019-11-24 03:39:53.595429",
  "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/base.py\", line 74, in __call__\n    return self._run(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/api_base.py\", line 59, in _run\n    f'HTTP status code \"{resp.status_code}\" while fetching {url}. '\nscenario_player.exceptions.legacy.RESTAPIStatusMismatchError: HTTP status code \"409\" while fetching http://127.0.0.1:40735/api/v1/payments/0x62083c80353Df771426D209eF578619EE68D5C7A/0x1c5E3b08b928A75585fe35286F8a3C89f417ba93. Expected 200: {\"errors\": \"Payment couldn't be completed because: there is no route available\"}"
}
```

what does the exception say:

1. Payment couldn't be completed because: there is no route available

2. Request 
`http://127.0.0.1:40735/api/v1/payments/0x62083c80353Df771426D209eF578619EE68D5C7A/0x1c5E3b08b928A75585fe35286F8a3C89f417ba93`

`0x62083c80353Df771426D209eF578619EE68D5C7A` : token address
`0x1c5E3b08b928A75585fe35286F8a3C89f417ba93` : node3

3. `- transfer: {from: 0, to: 3, amount: 1_000_000_000_000_000, expected_http_status: 200}`
line 82

SP 8

**Comment by agatsoh** [comment]
Initial findings have been entered

**Comment by Dominik1999** [comment]
Closed and merged into META Issue #5433
